### PR TITLE
dev-tictactoe Pull Request

### PR DESCRIPTION
**Pull request by FSolM**
Tic Tac Toe's pull request 👾

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-alert */
+/* eslint-disable import/extensions */
+
+import { gameController } from './lib/gameController.js';
+
+let nameP1 = '';
+let nameP2 = '';
+
+function playerMove (node, index) {
```
Comment (JamesOkunlade) [inline assets/js/main.js]: change to `es6 arrow function`

File: assets/js/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      deleteEventListeners();
+      document.getElementById('game-over').classList.remove('set-hidden');
+      alert('Game Over');
+      playAgain();
+    }
+    gameController.setCurrentPlayer();
+  }
+}
+
+function createEventListeners () {
```
Comment (JamesOkunlade) [inline assets/js/main.js]: change to `es6 arrow function`

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello, devs ✋ 😄 

- An error prevented your game from start. Please check your console for log and fix 

image

- Change all the classic functions in your `main.js` to `ES6 arrow functions`

- Add a descriptive README.md file to your project.

**Status: `Changes required`**

Happy coding! 😎

**Comment by diegoarvz4** [comment]
Hi James

To avoid the CORS policy error please try using the HTTP protocol with some local server. We recommend using the [VS Code's Live Version](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

Thanks a lot!

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hi devs ✋ 😎 

Good job converting to `ES6 arrow functions` 😄 

**Add a `live-server` as a dependency to your project and Update your README.md file to describe the installation process of your project.**

**Status: `Changes required`**

**Comment by FSolM** [comment]
For us to be able to add a live version through GitHub pages, the project needs to be pushed all the way to the master branch...

**Review by JamesOkunlade — APPROVED** [review]
### It's a 👍 for me.

**Go ahead and merge.**

**Good luck with your future projects.**

**Pull request merged**